static/usage/popover/presenting/inline-isopen/vue.md

```diff
@@ -0,0 +1,33 @@
+```html
+<template>
+  <ion-button @click="openPopover($event)">Click Me</ion-button>
```

Comment:
The rendering of this stackblitz looks different than the other ones.

| javascript | angular | react | vue |
| ---------- | ------- | ----- | ---- |
| ![image](https://user-images.githubusercontent.com/2721089/169117390-df16952f-7a08-4f2c-8c3c-9f4211ab6798.png) | ![image](https://user-images.githubusercontent.com/2721089/169117487-8054a558-3325-4f2f-94c1-ab5858ac99bf.png) | ![image](https://user-images.githubusercontent.com/2721089/169117459-3f61ed15-eef7-4f76-b92b-70144c393c8b.png) | ![image](https://user-images.githubusercontent.com/2721089/169117551-00c3c436-4186-4d75-9227-70c1fe933d1f.png) |